### ISSUE TITLE
allow for editing/deleting comments within digitization queue item view

### ISSUE THREAD

**Issue by yonyitz** [sciencehistory/kithe#47]
Please allow for editing/deleting comments within digitization queue item view.  Editing/deleting should only be permissible by author.

**Comment by jrochkind** [comment]
I think we decided that for now, just deleting is good?

@eddierubeiz we could work on this pairing maybe next week?

**Comment by yonyitz** [comment]
That's right.  Thanks for reminding me that deleting is fine in place of editing.  Thank you!
